website/source/docs/schedulers.html.md

```diff
@@ -6,6 +6,11 @@ description: |-
   Learn about Nomad's various schedulers.
 ---
 
+[Borg]: https://research.google.com/pubs/pub43438.html
+[preemption]: /docs/internals/scheduling/preemption.html
+[restart]: /docs/job-specification/restart.html
+[reschedule]: /docs/job-specification/reschedule.html
```

Comment:
I think we always put these at the bottom. (I didn't even realize you could put them anywhere... kind of handy really, but let's stay consistent.)

Reply:
We should make a note to visit this.  It's nice to have them as top matter.  Moving for consistency now.